### PR TITLE
Fix: not resuming issue

### PR DESCRIPTION
**Pull request by 1Lucas1apk**
## Changes

Update the client with clients.get and place the ready event within the code block

## Why 

To fix the problem 👽

## Checkmarks

- [x] The modified endpoints have been tested.
- [x] Used the same indentation as the rest of the project.
- [x] Still compatible with LavaLink clients.

## Additional information

You probably forgot to fix it so open the request

### CODE REVIEW

File: src/connection/handler.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ async function setupConnection(ws, req) {
 
   ws.on('close', (code, reason) => {
     debugLog('disconnect', 3, { code, reason })
-
+    client = clients.get(sessionId)
```
Comment (ThePedroo) [inline src/connection/handler.js:67]: Give a space between debugLog, client and the if statement

File: src/connection/handler.js [hunk 2]
```diff
@@ -124,13 +124,13 @@ async function setupConnection(ws, req) {
...
-    }
 
-    ws.send(JSON.stringify({
-      op: 'ready',
-      resumed: !!resumedClient,
-      sessionId: !!resumedClient ? sessionId : randomLetters(16)
-    }))
+      ws.send(JSON.stringify({
+        op: 'ready',
+        resumed: !!resumedClient,
```
Comment (ThePedroo) [inline src/connection/handler.js:130]: As it's inside the else now, it will always be `false`

File: src/connection/handler.js [hunk 3]
```diff
@@ -124,13 +124,13 @@ async function setupConnection(ws, req) {
...
+        sessionId: !!resumedClient ? sessionId : randomLetters(16)
```
Comment (ThePedroo) [inline src/connection/handler.js:131]: ditto. But with sessionId

**Review by ThePedroo — CHANGES_REQUESTED** [review]
LGTM, just some nitpicks

**Comment by 1Lucas1apk** [comment]
> LGTM, just some nitpicks

Review...

**Review by ThePedroo — APPROVED** [review]
Perfect!

**Pull request merged**